make_prg/subcommands/from_msa.py

```diff
@@ -28,6 +28,17 @@ def register_parser(subparsers):
             "Multiple sequence alignment file or a directory containing such files"
         ),
     )
+    subparser_msa.add_argument(
```

Comment:
Accepting multiple suffixes would be a better approach for completeness?

Reply:
Could use the same approach that we use in snakefmt with an include/exclude regex? https://github.com/snakemake/snakefmt/blob/d233dfa958479f8ee3b4a341f6ccc3fb8ae9ab78/snakefmt/snakefmt.py#L41-L76

Reply:
I can't see a better solution than the include/exclude regex, will implement it, thanks!

Reply:
Logged this in an issue: https://github.com/iqbal-lab-org/make_prg/issues/39 and I am moving on